src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,10 +1,8 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.dataStructures.IntArraySTack;
-
 public class FIFORhymer extends DefaultCountingOutRhymer {
 
-    public final IntArraySTack temp = new IntArraySTack();
+    private final DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
```

Comment:
Reverting is not a good solution. Use the linked list implementation (which does not have a size limit)

Reply:
Try doing 2.2.8 again. Use IntLinkedList